.github/workflows/bootstrap.sh

```diff
@@ -104,6 +106,8 @@ echo "archive_site_local https://packages-private.macports.org/:tbz2" | sudo tee
 # preferred_hosts has no effect on archive_site_local
 # See https://trac.macports.org/ticket/57720
 #echo "preferred_hosts packages.macports.org" | sudo tee -a /opt/local/etc/macports/macports.conf >/dev/null
+# Temporarily prefer mirror: https://trac.macports.org/ticket/68563
+echo "preferred_hosts mirror.fcix.net" | sudo tee -a /opt/local/etc/macports/macports.conf >/dev/null
```

Comment:
This part shouldn't be needed with the more general change to the mirror list in the ports tree.